### ISSUE TITLE
Form field descriptions do not have sufficient contrast

### ISSUE THREAD

**Issue by pinussilvestrus** [bpmn-io/form-js#334]
__Describe the Bug__

The description elements of a form-js form field do not have sufficient contrast against white background [according to WCAG 2.1 (AA)](https://dequeuniversity.com/rules/axe/4.4/color-contrast).

![image](https://user-images.githubusercontent.com/9433996/191196893-c77e8fb5-a363-4644-9c39-6139b840d13e.png)

__Steps to Reproduce__

1. Open https://demo.bpmn.io/form/s/start
2. Inspect a form field description, e.g. via Chrome Developer Tools
3. See the warning on non-sufficient contrast

__Expected Behavior__

Increase the contrast of `.fjs-form-field-description` elements. We should also update our [basic a11y tests](https://github.com/bpmn-io/form-js/blob/develop/packages/form-js-viewer/test/spec/render/components/form-fields/Textfield.spec.js#L205) to cover this one (adding descriptions). 

__Environment__

- Host (Browser/Node version), if applicable: Chrome 105.0.5195.125
- OS: macOS 11.6.5
- Library version: `master`

**Comment by pinussilvestrus** [comment]
I added a test case that showcases this issue: https://github.com/bpmn-io/form-js/commit/e85b69516a68e69c7199512a570974c06b5cb56b.

This was initially discovered due to [a failing test case](https://github.com/camunda/form-playground/actions/runs/3088227838/jobs/4994458052#step:6:142) in `@camunda/form-playground`.

**Comment by pinussilvestrus** [comment]
In the properties panel, we use `--color-grey-225-10-35` for any description. We could use the same for form field descriptions to fix the issue.

_Properties panel_
![image](https://user-images.githubusercontent.com/9433996/191199690-02d1c460-ad55-46a2-9772-df1b86616e78.png)

_Form field_
![image](https://user-images.githubusercontent.com/9433996/191199843-f11dc79e-af34-4f6c-83d4-ed0ad4c578e7.png)

**Comment by pinussilvestrus** [comment]
Closed via https://github.com/bpmn-io/form-js/issues/334

**Comment by pinussilvestrus** [comment]
Just released this minor fix via [`0.9.1`](https://github.com/bpmn-io/form-js/commit/9fb8cf4a9bffe209ad7f76df8a9d764e1f782803).
